.github/workflows/chart-lint.yaml

```
@@ -0,0 +1,33 @@
+name: Test Helm Chart
+
+on: push
+
+jobs:
+  kubeval:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.1
+      - name: template
+        run: |
+          CHARTS=charts/localstack
+          TMP=./tmp/templates
+
+          mkdir -p $TMP
+
+          cd $CHARTS
+          helm dependency update
+          cd -
```

You can use pushd/popd here (doesn't really make a difference in this case.

```suggestion
          pushd $CHARTS
          helm dependency update
          popd
```

---

I liked the `tee` suggestion :) this one seems more like personal preference.

---

Indeed :) Sounds good to me!

---

Thanks @nhomble @ls-jad-elkik . nitpick: Another possible option would be to use a subshell:
```
(cd $CHARTS; helm dependency update)
```
But I think what we have is good as well. 👍